Doc/library/fractions.rst

```diff
@@ -94,6 +94,13 @@ another rational number, or from a string.
       Denominator of the Fraction in lowest term.
 
 
+   .. method:: as_integer_ratio()
+
+      Return a tuple of integers, whose ratio is exactly equal
```

Comment:
Drop "exactly"

Reply:
@gvanrossum To some degree, I think that mentioning "exactly" might help readers to understand the advantage of using an integer ratio when they need more precision. However, there might be a better way to mention this.

Reply:
Thanks. I'll drop this "exactly".  The wording was take from the docstring for *float.as_integer_ratio()* but in makes a lot more sense.